cpp/dolfinx/fem/Function.h

```diff
@@ -225,6 +228,15 @@ class Function
     fem::interpolate(*this, fx, cells);
   }
 
+  /// Interpolate an Expression (based on ufl)
+  /// @param[in] expr The function to be interpolated.
+  /// @param[in] cells The cells (local to process) to interpolate into
+  void interpolate(const Expression<T>& expr,
```

Comment:
I'd remove this. It is confusing and should be executed with dolfinx.Expression.interpolate directly.

Reply:
See my comment above. As long as expression need to take in the interpolation points, I thinks the interface becomes unnecessary explicit.